statics/js/components/graph-layout.js

```diff
@@ -9,7 +9,15 @@ var TopologyGraphLayout = function(vm, selector) {
 
   this.queue = new Queue();
   this.queue.await(function() {
-    if (self.invalid) self.update();
+    if (self.invalid) {
+      if (self._autoExpand) {
```

Comment:
could you declare it in the "constructor", set as false.
You never reset the autoExpand variable meaning that it is going to expand for each events, same thing for the zoomFit. So If a user change the zoom it will be changed. I would prefer to have it just once.

Reply:
the intent was that the view keeps adjusting automatically as new updates arrive ... we could (preferably is a separate PR) add a toggle button ("Automatic Expand") which can be set to on by adding the ?Expand option, but can later be disabled by user (only after disabling one could switch back to manual Expand, Zoom, etc.

Reply:
looks OK to me